contrib/resources/translations/README.md

```diff
@@ -0,0 +1,98 @@
+
+# General translation workflow
+
+## Preparing file with English strings
+
+First, you need an up to date file with English strings. The one present in this
+directory is most likely outdated; to generate the current one either execute:
```

Comment:
(small suggestion)

Be sure you have the latest English messages file, `en.lng`, by running:

Reply:
Changed, but I have kept a warning that the supplied file is most likely outdated.